src/devices/ExplorerHat/samples/ExplorerHat.BasicSample/ExplorerHat.BasicSample.csproj

```diff
@@ -0,0 +1,17 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <ApplicationIcon />
+    <StartupObject />
```

Comment:
what are those 2 above? Can you match csproj with other csproj-es (including the name)

Reply:
to clarify when there is only one sample you should not create subfolder and name should be `ProjectName.Sample` - please refer to other projects. Same for the content of csproj

Reply:
Done